vividus-plugin-web-app/src/test/java/org/vividus/ui/web/action/search/VisibilityTests.java

```diff
@@ -27,6 +27,7 @@
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.Arguments;
 import org.junit.jupiter.params.provider.MethodSource;
+import org.vividus.ui.action.search.Visibility;
 
 class VisibilityTests
```

Comment:
package for this class should be updated to `org.vividus.ui.action.search` and this class should be moved to `vividus-extension-selenium`